### ISSUE TITLE
numpy np.bool8 is deprecated but used by plotly 5.11.0.

### ISSUE THREAD

**Issue by ievgennaida** [plotly/plotly.py#4010]
np.bool8 is giving deprecated warning.

```
env\lib\site-packages\plotly\express\imshow_utils.py:24
  \plotly\express\imshow_utils.py:24: DeprecationWarning: `np.bool8` is a deprecated alias for `np.bool_`.  (Deprecated NumPy 1.24)
    np.bool8: (False, True),
```

numpy==1.24.1
plotly==5.11.0

**Comment by AaronStiff** [comment]
Fixed in `plotly 5.12.0` and up, specifically by #3997

**Comment by nicolaskruchten** [comment]
Ah yes, thanks, i forgot to close this issue :)
